index.js

```diff
@@ -18,7 +18,7 @@ const userRoutes = require("./routes/users");
 const feedRoutes = require("./routes/feeds");
 const apiRoutes = require("./routes/apis");
 
-const dbUrl = process.env.DB_URL || "mongodb://localhost:27017/Meddit"; // 27017 is the default mongodb port
+const dbUrl = process.env.DB_URL;// || "mongodb://localhost:27017/Meddit"; // 27017 is the default mongodb port
```

Comment:
I suggest to not comment out the localhost environment
The flow goes like this:
1. It checks in your .env file whether you have DB_URL set or not, if not, it will run in local environment
2. If you have, it runs on the mongodb remote database

So if you want to run on local environment, best is just comment the DB_URL variable in .env temporarily, so it wouldn't ruin the git files